### PR TITLE
Fixed: Type error, reported from the bitmessage chan

### PR DESCRIPTION
**Pull request by coffeedogs**
Fixed: Type error, reported from the bitmessage chan

Untested.

### CODE REVIEW

File: src/network/bmproto.py [hunk 1]
```diff
@@ -334,7 +334,7 @@ def bm_command_object(self):
         self.object = BMObject(nonce, expiresTime, objectType, version, streamNumber, self.payload, self.payloadOffset)
 
         if len(self.payload) - self.payloadOffset > BMProto.maxObjectPayloadSize:
-            logger.info('The payload length of this object is too large (%s bytes). Ignoring it.' % len(self.payload) - self.payloadOffset)
+            logger.info('The payload length of this object is too large (%d bytes). Ignoring it.' % len(self.payload) - self.payloadOffset)
```
Comment (PeterSurda) [inline src/network/bmproto.py]: doesn't it have to be in brackets, like `(len(self.payload) - self.payloadOffset))`?

Comment (coffeedogs) [reply src/network/bmproto.py]: Indeed, updated.

File: src/network/bmproto.py [hunk 2]
```diff
@@ -334,7 +334,7 @@ def bm_command_object(self):
         self.object = BMObject(nonce, expiresTime, objectType, version, streamNumber, self.payload, self.payloadOffset)
 
         if len(self.payload) - self.payloadOffset > BMProto.maxObjectPayloadSize:
-            logger.info('The payload length of this object is too large (%s bytes). Ignoring it.' % len(self.payload) - self.payloadOffset)
+            logger.info('The payload length of this object is too large (%d bytes). Ignoring it.' % (len(self.payload) - self.payloadOffset))
```
Comment (g1itch) [inline src/network/bmproto.py]:
```python
payload_size = len(self.payload) - self.payloadOffset
if payload_size > BMProto.maxObjectPayloadSize:
    logger.info('The payload length of this object is too large (%s bytes). Ignoring it.', payload_size)
```

Please remove percent. `payload_size` or `len(self.payload) - self.payloadOffset` should be second argument of `logger.info()`.

Comment (g1itch) [reply src/network/bmproto.py]: I remember landscape.io marked such mistakes.

Comment (PeterSurda) [reply src/network/bmproto.py]: I liked landscape.io but the service didn't have a reliable uptime.

Comment (coffeedogs) [reply src/network/bmproto.py]: While I agree, this was deliberately the smallest change I could make to make the fix. Can we start an issue to remove all % formatting in log statements throughout the project?

**Pull request merged**